vpc/tool/container2/setup_container_linux.go

```diff
@@ -127,7 +127,7 @@ func DoSetupContainer(ctx context.Context, netnsfd int, bandwidth, ceil uint64,
 	return configureLink(ctx, nsHandle, newLink, bandwidth, ceil, mtu, allocation)
 }
 
-func addIPv4AddressAndRoute(ctx context.Context, nsHandle *netlink.Handle, link netlink.Link, mtu *int, address *vpcapi.UsableAddress) error {
+func addIPv4AddressAndRoute(ctx context.Context, nsHandle *netlink.Handle, link netlink.Link, mtu *int, address *vpcapi.UsableAddress, routes []*vpcapi.AssignIPResponseV3_Route) error {
```

Comment:
We already have the (link) mtu here, do we need the whole routes[] complexity, or can we get away with a routeMtu int to use here?

In fact, it looked like previously if `mtu != nil` we *were* setting the mtu on the default route. Could we just start actually passing the mtu into this function if jumbo?


I guess it looks like this PR is 2 things: jumbo support and extending the api to handle multiple additional routes. Is the extra routes something we will eventually need?

Reply:
I guess this PR is really moving the jumbo frame logic to the server, from the individual agents.

Reply:
If the server side doesn't send down Jumbo routes, it'll fall back to the local settings. Otherwise, (hopefully soon) we can add logic to have a greenlist on the titus vpc service about what other VPCs / subnets to send jumbo traffic to.